A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/cruise.FLT

```diff
@@ -198,6 +198,7 @@ A32NX_OVHD_HYD_BLUEPUMP_OVRD=0
 A32NX_OVHD_HYD_LEAK_MEASUREMENT_G=1
 A32NX_OVHD_HYD_LEAK_MEASUREMENT_B=1
 A32NX_OVHD_HYD_LEAK_MEASUREMENT_Y=1
+A32NX_INITIAL_FLIGHT_PHASE = 3
```

Comment:
I choose 3 (= climb) here because the transitions are a bit sloppily implemented by Asobo. 4 (= cruise) sometimes ends up in 5 (= descent) based on the barometic height the aircraft spawns at (5000 AGL, but different barometric heights).
I'll clean those transitions up in a future PR and then we can revisit this.